tox.ini

```diff
@@ -10,10 +12,10 @@ commands =
     zope-testrunner --test-path=src
 
 [testenv:coverage]
-basepython = python2.7
+basepython = python3.7
 commands =
     coverage run -m zope.testrunner --test-path=src {posargs:--auto-color -vv}
-    coverage report --fail-under=92
+    coverage report --show-missing --fail-under=93
 deps = coverage
 
 [testenv:flake8]
```

Comment:
This could be also run with Python 3.7

Reply:
okay.

Reply:
This even led to a still existing SyntaxError in `src/z3c/__init__.py`.